spacy_llm/tasks/util/span.py

```diff
@@ -35,6 +36,7 @@ def __init__(
         self._alignment_mode = alignment_mode
         self._case_sensitive_matching = case_sensitive_matching
         self._single_match = single_match
+        self._uses_quoted_strings = "quoted strings" in template
```

Comment:
This is hacky. The motivation is that parsing comma-delimited lists of either quoted or unquoted strings was cumbersome - parsing is easier if we know whether to expect quoted or unquoted strings, hence this check. Putting this up because I'm looking for better ideas (either to check whether quoted or unquoted strings are used or a unified approach to parsing the response).